src/v1/providers/firestore.ts

```diff
@@ -175,7 +163,6 @@ function changeConstructor(raw: Event) {
 }
 
 export class DocumentBuilder<Path extends string> {
```

Comment:
Same thing with DocumentBuilder. This class is only used in this file. Remove `export` or `@internal`?

Reply:
Same thing - this is probably exposed to customer and must be exported and not be internal. (I could be wrong though).

Reply:
SGTM!